<?xml version="1.0" encoding="UTF-8"?>
<!--
This file is based on code from the QGIS plugin "Deepness"
(https://github.com/PUTvision/qgis-plugin-deepness, licensed under the Apache License, Version 2.0).

Original copyright: 2021-2025 PUT vision, Przemyslaw Aszkowski
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.

Modifications have been made by Tanja Kempen / TrailScan in 2025:
 - Adapted for use in the TrailScan QGIS plugin
-->
<ui version="4.0">
 <class>PackagesInstallerDialog</class>
 <widget class="QDialog" name="PackagesInstallerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrailScan - Packages Installer Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTextBrowser" name="textBrowser_log"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_install_packages">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Install packages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_diagnostics">
       <property name="text">
        <string>Show Diagnostics</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>Test and Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
